.gitignore

```diff
@@ -11,11 +11,10 @@ hack/tools/bin
 *.test
 
```

Comment:
Note to myself:
* [ ] Improve corresponding docs (test/e2e/config/README.md + release tasks doc)